MAUI/Barcode/customization.md

```diff
@@ -0,0 +1,364 @@
+title: Customization in .NET MAUI Barcodes (SfBarcodeGenerator) | Syncfusion
+description: Learn here all about Customization of Syncfusion .NET MAUI Barcodes(SfBarcodeGenerator) control with key features and more.
+platform: maui
+control: SfBarcodeGenerator
+documentation: ug
+---
+
+# Customization in .NET MAUI Barcodes (SfBarcodeGenerator)
+
+## Text customization
+
+### Displaying input value
+
+The provided input value of the barcode can be displayed by enabling its `ShowText` property. By default, the `ShowText` is false.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="4" %}
+
+        <barcode:SfBarcodeGenerator Value="12634388927" 
+                                    HeightRequest="150"
+                                    WidthRequest="300" 
+                                    ShowText="True"/>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="5" %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 300;
+        barcode.Value = "12634388927";
+        barcode.ShowText = true;
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Customization Show Text](images/customization/maui-customization-show-text.png)
+
+### Text style customization
+
+The style of the text can be customized with the `TextStyle` property of the barcode generator.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="6 7 8 9" %}
+
+            <barcode:SfBarcodeGenerator Value="12634388927" 
+                                        HeightRequest="150"
+                                        WidthRequest="300" 
+                                        ShowText="True">
+                <barcode:SfBarcodeGenerator.TextStyle>
+                    <barcode:BarcodeTextStyle FontAttributes="Italic" 
+                                              FontSize="16" 
+                                              FontFamily="Times" 
+                                              TextColor="Red"/>
+                </barcode:SfBarcodeGenerator.TextStyle>
+            </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="6 7 8 9 10 11 12" %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 300;
+        barcode.Value = "12634388927";
+        barcode.ShowText = true;
+        barcode.TextStyle = new BarcodeTextStyle()
+        {
+            FontAttributes = FontAttributes.Italic,
+            FontFamily = "Time",
+            FontSize = 16,
+            TextColor = Colors.Red
+        };
+
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Text Customization](images/customization/maui-text-customization.png)
+
+### Text spacing
+
+The space between the text and the barcode can be controlled by the `TextSpacing` property of barcode generator. By default, the value of `TextSpacing` is 2.
```

Comment:
The space between the text and the barcode can be controlled by using the `TextSpacing` property of the barcode generator.

Reply:
Changes updated.